doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -238,10 +238,13 @@ example, for the above test ID, the link is - http://pulpito.front.sepia.ceph.co
 
 Re-running Tests
 ----------------
-You can pass ``--rerun`` option, with test ID as an argument to it, to
-``teuthology-suite`` command. Generally, this is useful in cases where teuthology test
-batch has some failed/dead jobs that we might want to retrigger. We can trigger
-jobs based on their status using::
+
+The ``teuthology-suite`` command has a ``--rerun`` option, which allows you to
+re-run tests. This is handy when your test has failed or is dead. The
+``--rerun`` option takes the name of a test as an argument, as you can see in
```

Comment:
the name of a teuthology run as an argument

Reply:
Roger that, Neha. Changed and pushed back to you for review.